docs/developers/configuration.md

```diff
@@ -0,0 +1,173 @@
+---
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Configuration
+Celeborn configuration is separated into static and dynamic configuration. Static configuration `CelebornConf` is specified in 
+default configuration file `$CELEBORN_HOME/conf/celeborn-defaults.conf`. Meanwhile, the dynamic configuration overrides 
```

Comment:
## Static Configuration
Static configuration `CelebornConf` is specified in default configuration file `$CELEBORN_HOME/conf/celeborn-defaults.conf`. 

## Dynamic Configuration
The dynamic configuration overrides the static configuration at runtime of `Master` and `Worker`.